tests/test_unit/test_features_generators.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+from chemprop.features import features_generators
+from rdkit import Chem
+from typing import Sequence
+
+TEST_SMILES_STRING = "C"
+TEST_MOLECULE_OBJECT = Chem.MolFromSmiles(TEST_SMILES_STRING)
+
+TEST_MULTIMOLECULE_LIST = [
+    [TEST_SMILES_STRING, TEST_SMILES_STRING],
+    [TEST_SMILES_STRING, TEST_SMILES_STRING],
+]
+
+MORGAN_RADIUS = 2
+MORGAN_NUM_BITS = 2048
+
+NUM_RDKIT_FEATURES = 200
+
+
+# Ensure the output dimensionality of each generator is correct
+class TestFeatureGenerators:
+    class TestOutputDimensionality:
+        @pytest.mark.parametrize(
+            "mol_data, radius, num_bits",
+            [
+                (TEST_SMILES_STRING, MORGAN_RADIUS, MORGAN_NUM_BITS),
+                (TEST_MOLECULE_OBJECT, MORGAN_RADIUS, MORGAN_NUM_BITS),
+                (TEST_MULTIMOLECULE_LIST, MORGAN_RADIUS, MORGAN_NUM_BITS),
+            ],
+        )
```

Comment:
you can accomplish the same thing with pytest `fixture`s to avoid repeating the same `parameterize` statements

Reply:
Will do the same in the integration tests file (which will be renamed, it having the same name as the unit test file is breaking the test runner). Forgot to switch the test parameterization from paramterized to pytest.mark.parametrize in that file.